src/types/types.ts

```diff
@@ -851,6 +851,9 @@ export interface GetOwnersForNftResponse {
 export interface GetOwnersForContractResponse {
   /** An array of owner addresses for the provided contract address */
   owners: string[];
+
+  /** Total count of unique owners */
+  totalCount: number;
```

Comment:
this should be optional right? 

Can you also add a comment here that to get this field you have to set `{@link GetOwnersForContractOptions.includeCount`} to true?